src/common/core/setcore.js

```diff
@@ -278,6 +282,20 @@ define(['common/core/CoreAssert', 'common/core/constants'], function (ASSERT, CO
                 self.createSet(node, setName);
             }
         }
+
+        function loadNodesOfPaths(root, paths) {
+            var nodes = [],
+                i,
+                rootHash = self.getHash(root);
+
+            return TASYNC.call(function () {
+                for (i = 0; i < paths.length; i += 1) {
+                    nodes[i] = self.loadByPath(root, paths[i]);
+                }
+                return TASYNC.lift(nodes);
```

Comment:
Couldn't this suffer from the same issue as with children?
That they have been removed and loadByPath returns null?
I think it doesn't since the members are stored in the ovr and cleaned at deletion.. But just checking..